spec/requests/post911_gi_bill_status_request_spec.rb

```diff
@@ -19,47 +20,15 @@
 
     after { Timecop.return }
 
-    context 'with a valid evss response' do
-      it 'GET /v0/post911_gi_bill_status returns proper json' do
-        VCR.use_cassette('evss/gi_bill_status/gi_bill_status') do
-          get v0_post911_gi_bill_status_url, params: nil
+    context 'with a 200 response' do
+      it 'GET /v1/post911_gi_bill_status returns proper json' do
+        VCR.use_cassette('lighthouse/benefits_education/gi_bill_status/200_response') do
+          get v1_post911_gi_bill_status_url, params: nil
           expect(response).to match_response_schema('post911_gi_bill_status')
           assert_response :success
         end
       end
     end
-
-    # TODO(AJD): this use case happens, 500 status but unauthorized message
-    # check with evss that they shouldn't be returning 403 instead
-    context 'with an 500 unauthorized response' do
-      it 'returns a forbidden response' do
-        VCR.use_cassette('evss/gi_bill_status/unauthorized') do
-          get v0_post911_gi_bill_status_url, params: nil
-          expect(response).to have_http_status(:forbidden)
-        end
-      end
-    end
-
-    # TODO: - is this a real scenario?
-    context 'with a 403 response' do
-      it 'returns a forbidden response' do
-        VCR.use_cassette('evss/gi_bill_status/gi_bill_status_403') do
-          get v0_post911_gi_bill_status_url, params: nil
-          expect(response).to have_http_status(:forbidden)
-        end
-      end
-    end
-
-    # a 500 and does not contain one of the errors defined in:
-    # https://csraciapp6.evss.srarad.com/wss-education-services-web/swagger-ui/ext-docs/education-error-keys.html
-    context 'with an undefined 500 evss response' do
-      it 'returns internal server error' do
-        VCR.use_cassette('evss/gi_bill_status/gi_bill_status_500') do
-          get v0_post911_gi_bill_status_url, params: nil
-          expect(response).to have_http_status(:internal_server_error)
```

Comment:
Old EVSS way of doing things was to always return a 200 response, with error details in the response data.  Ruby code had to interpret those responses, and raise errors accordingly.  Lighthouse returns "real" http responses, so no need to test for a 500, 403 etc.